tests/regression/test_interpolate.py

```diff
@@ -107,6 +124,48 @@ def test_compound_expression():
     assert np.allclose(g.dat.data, h.dat.data)
 
 
+# Requires the relevant FInAT or FIAT duals to be defined
+@pytest.mark.xfail(raises=NotImplementedError)
```

Comment:
Note this and the below test xfails are not regressions - they don't work on master either. See ##1966